dimagi/locale/en_US/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,38 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Cool. I wasn't sure whether to add this for when dimagi-utils is used independently of HQ, so I made sure to put it in a separate commit. I'll drop it.